package.json

```diff
@@ -22,5 +22,9 @@
   "devDependencies": {
     "fs-extra": "11.3.0",
     "node-fetch": "^3.3.2"
+  },
```

Comment:
THis should not be required and wont affect your workspace as this file has no control over it.